fix(xo-web/backups): scheduled health check is available to enterprise (#6257)
vatesfr/xen-orchestra: CHANGELOG.unreleased.md

- [load-balancer] Fix density mode failing to shutdown hosts (PR [#6253](https://github.com/vatesfr/xen-orchestra/pull/6253))
- [Health] Make "Too many snapshots" table sortable by number of snapshots (PR [#6255](https://github.com/vatesfr/xen-orchestra/pull/6255))
- [Remote] Show complete errors instead of only a potentially missing message (PR [#6216](https://github.com/vatesfr/xen-orchestra/pull/6216))

> - [Backup] Scheduled health check of a backup is available from the enterprise level (PR [#6257](https://github.com/vatesfr/xen-orchestra/pull/6257))

Reviewer: ```suggestion
- [Backup] Scheduled Health Check should be available in Enterprise Edition (PR [#6257](https://github.com/vatesfr/xen-orchestra/pull/6257))
```
